style.css

```diff
@@ -0,0 +1,1131 @@
+html {
+	box-sizing: border-box;
+}
+
+*, *:before, *:after {
+	box-sizing: inherit;
+}
+	
+body {
+	min-height: 100vh;
+	background: #fff;
+	font-size: 12px;
+	line-height: 11px;
+	font-family: 'GothaPro';
+	color: #000;
+	margin: 0;
+}
+
+.container {
+	max-width: 1030px;
+	margin: 0 auto;
+}
+
+.site-navigation,
+.user-navigation,
+.catalog-list,
+.account-list,
+.customer-list,
+.miscellaneous-list,
+.main-footer__social {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+}
+
+.main-header__nav-wrapper {
+	display: flex;
+	padding-top: 33px;
+	padding-bottom: 28px;
+}
+
+.main-header__burger {
+	display: none;
+}
+
+.adapted-logo {
+	display: none;
+}
+
+.main-header__logo {
+	margin-right: 45px;
+}
+
+.site-navigation {
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	text-align: justify;
+}
+
+.site-navigation__item {
+	margin-right: 35px;
+	text-transform: uppercase;
+}
+
+ .site-navigation__item:last-child {
+ 	margin-right: 0;
+ }
+
+
+.link-item {
+	text-decoration: none;
+	color: #000;
+	font-family: 'GothaPro';
+	font-weight: 700;
+}
+
+.link-item:hover {
+	color: #b48484;
+	opacity: .8;
+}
+
+.user-navigation {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	text-align: justify;
+	margin-left: auto;
+}
+
+.user-navigation .link-item {
+	margin-right: 35px;
+	text-transform: uppercase;
+}
+
+.user-navigation .link-item:last-child {
+	margin-right: 0
+}
+
+.link-item_img {
+	position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GerKateAxion/pullRequest?prid=5991795)